resources/assets/less/bem/beatmapset-stats.less

```diff
@@ -2,82 +2,27 @@
 // See the LICENCE file in the repository root for full licence text.
 
 .beatmapset-stats {
-  color: #fff;
-  font-size: 12px;
+  display: grid;
+  grid-template-columns: auto auto 1fr;
```

Comment:
better with a minmax for the numbers so they don't move around when switching difficulties